### PR TITLE
Added default user agent

### PR DESCRIPTION
**Pull request by Pekket**

### CODE REVIEW

File: src/main/kotlin/org/rescado/server/constant/SecurityConstants.kt [hunk 1]
```diff
@@ -16,6 +16,7 @@ object SecurityConstants {
     const val SECRET = "n2r5u8x/A%D*G-KaPdSgVkYp3s6v9y?/B&E(H+MbQeThWmZq4t7w!z%C*F-J@NcRf" // sorry hackers
 
     const val DEVICE_HEADER = "User-Device"
+    const val DEFAULT_USER_AGENT = "Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10.9.8; rv:15.6) Gecko/20100101 Firefox/15.6.6"
```
Comment (Qrivi) [inline src/main/kotlin/org/rescado/server/constant/SecurityConstants.kt]: Remove `DEFAULT_` prefix because it's not a placeholder but a fixed value.

**Comment by Qrivi** [comment]
Perhaps also add and implement a `REQUEST_TIMEOUT` after which the check always fails to prevent hanging when passed a URL that does not resolve?

Does not have to be in the same PR. Will approve if you remove the prefix. 👽 But perhaps it's just a setter? #quickwin

File: src/main/kotlin/org/rescado/server/service/ImageService.kt [hunk 2]
```diff
@@ -43,7 +45,8 @@ class ImageService(
         var contentType = ""
         return try {
             val url = URL(reference)
-            val con = url.openConnection()
+            val con = (url.openConnection() as HttpURLConnection)
```
Comment (Qrivi) [inline src/main/kotlin/org/rescado/server/service/ImageService.kt]: Is this a safe cast? Does this work with https URLs? Perhaps safer to check the string and if it starts with https, cast to `HttpsURLConnection`?

File: src/main/kotlin/org/rescado/server/constant/SecurityConstants.kt (hunk 1 above)
Comment (Qrivi) [reply src/main/kotlin/org/rescado/server/constant/SecurityConstants.kt]: This. idky when I commented with my phone it did not mark this as a "conversation". Also fyi the user agent you picked is from an OS that is 10 years old. Perhaps pick [a more recent one](https://www.google.com/search?q=what+is+my+user+agent) to fly even lower under the radar. 🥷

Unrelated but also please remove empty line on line 8. 😃

File: src/main/kotlin/org/rescado/server/service/ImageService.kt (hunk 2 above)
Comment (Qrivi) [reply src/main/kotlin/org/rescado/server/service/ImageService.kt]: Still wish I knew if this cast caused issues. 🙂 But I'll test it myself then.

**Review by Qrivi — APPROVED** [review]

File: src/main/kotlin/org/rescado/server/service/ImageService.kt (hunk 2 above)
Comment (Pekket) [reply src/main/kotlin/org/rescado/server/service/ImageService.kt]: The cast was removed in timeout commit

**Pull request merged**